conferences/migrations/2_create_ticketing_tables.up.sql

```diff
@@ -0,0 +1,65 @@
+CREATE TABLE slot_claim (
+    id BIGSERIAL PRIMARY KEY,
+    conference_slot_id BIGINT, 
+    ticket_id VARCHAR(100) CONSTrAINT ticket_id_is_unique UNIQUE,

```

Comment:
Could this instead be just:
```suggestion
    ticket_id TEXT UNIQUE,
```

VARCHAR doesn't provide any efficencies over TEXT in postgres (https://www.postgresql.org/docs/current/datatype-character.html). Is there a reason we need it to be constrained at 100 bytes?

Reply:
It was intended more for human readers than for the db, as a way to give an idea of the data expected there which in this case it is a string containing a representation of a uuid, glad go change it though i am not especially tied to any of this code